### DISCUSSION TITLE
music.163.com is not recognized

### DISCUSSION THREAD

**Discussion by recdown** [aclap-dev/video-downloadhelper#272] [Help and Q&A]
### Your question

music.163.com is not recognized

### Addon & CoApp version and details

music.163.com is not recognized

**Comment by paulrouget** [comment]
Please please please, there's a big warning when you open a question. Please pay attention.

Do precisely what's described here: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F
